src/Sarif.FunctionalTests/v2/ConverterTestData/FortifyFpr/FortifyTest.fpr.sarif

```diff
@@ -0,0 +1,377 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
+  "version": "2.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "HP Fortify Static Code Analyzer"
```

Comment:
>Fortify [](start = 20, length = 7)

Can you please look to see whether we can put all the Fortify files into a single 'Fortify' directory? Or is this complicated to do due to patterns around file extensions, etc., for a directory that has a single tool but multiple output formats?

Reply:
Yes, the UpdateBaselines script currently looks for input files with one specific extension. It seems sensible to separate them, since there are two completely independent converters.

---
In reply to: [207673935](https://github.com/Microsoft/sarif-sdk/pull/981#discussion_r207673935) [](ancestors = 207673935)

Reply:
Got it. Let's leave this as is. 

---
In reply to: [207675864](https://github.com/Microsoft/sarif-sdk/pull/981#discussion_r207675864) [](ancestors = 207675864,207673935)